README.md

```diff
@@ -1,40 +1,60 @@
 # Core APIs NestJS Service Template
 
 MadeiraMadeira boilerplate project to build scalable, testable and high performance Node.js microservices. 
- 
+
+## Service Architecture
+This service make part of the project <project name>.
+![Swagger OpenApi](docs/service-arch.png)
+> Obs: image to describe the service architecture or project architecture.
+
+## General Kong Routes Architecture
+Kong routes map.
+
+![Swagger OpenApi](docs/service-routes.png)
+> Obs: image to describe the service routes or project routes.
+
 ## Stack
 
 - [NestJS](https://github.com/nestjs/nest): a progressive Node.js + TypeScript framework based on dependency injection;
 - [gRPC](https://grpc.io): a Remote Procedure Call framework that provides high performance communication between microservices.
+- [Swagger + OpenAPI](https://swagger.io/specification/): an OpenAPI definition can then be used by documentation generation tools to display the API, code generation tools to generate servers and clients in various programming languages, testing tools, and many other use cases. 
+
+## Swagger Docs
+The current documentation of the project can be found on url `/docs`.
+![Swagger OpenApi](docs/openapi.png)
+
+## Release notes
+In this section contains the release notes of the project.
```

Comment:
Sounds more natural:

"This section describes the project release notes"

or

"This sections contains the project release notes".